concent_api/core/tests/test_unit_validation.py

```diff
@@ -266,11 +272,21 @@ def test_that_if_messages_are_signed_by_different_requestors_method_should_retur
             self.task_to_compute_2,
             subtask_2_signed_by=DIFFERENT_REQUESTOR_PRIVATE_KEY,
         )
-        result = are_subtask_results_accepted_messages_signed_by_the_same_requestor(subtask_results_accepted_list)
+        result = are_subtask_results_accepted_messages_properly_signed(subtask_results_accepted_list)
         assert_that(result).is_false()
 
+    def test_that_if_messages_are_signed_by_concent_method_should_return_true(self):
```

Comment:
Maybe also for case, when one message is signed by concent and second by requestor?

Reply:
ok, I will add